server/game/Clocks/ChessClock.js

```diff
@@ -2,27 +2,68 @@ const Clock = require('./Clock');
 
 class ChessClock extends Clock {
     constructor(player, time) {
-        super(player, time);
+        super(player, time, 5);
         this.mode = 'stop';
         this.name = 'Chess Clock';
     }
 
+    pause() {
+        this.stop();
+    }
+
+    restart() {
+        this.start();
+    }
+
+    reset() {
+        this.stop();
+    }
+
     start() {
-        this.mode = 'down';
-        super.start();
+        if(!this.manuallyPaused) {
+            if(this.mode !== 'down') {
+                this.mode = 'down';
+                super.start();
+            }
+        }
     }
 
     stop() {
         super.stop();
         this.mode = 'stop';
     }
 
+    opponentStart() {
+
+    }
```

Comment:
This is used in other clocks (not sure what it does), but it was causing issues with the chess clocks.  This is an empty override to remove that functionality only for chess clocks.